src/Clients/ChannelClient.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using StreamChat.Models;
+using StreamChat.Rest;
+
+namespace StreamChat.Clients
+{
+    public partial class ChannelClient : ClientBase, IChannelClient
+    {
+        internal ChannelClient(IRestClient client) : base(client)
+        {
+        }
+
+        public async Task<QueryChannelResponse> QueryChannelsAsync(QueryChannelsOptions opts)
+            => await ExecuteRequestAsync<QueryChannelResponse>("channels",
+                HttpMethod.GET,
```

Comment:
let's use post version, it can limit in url size since sometime query is too big

Reply:
✅